### ISSUE TITLE
What about the Lumia 950 XL GPU driver?

### ISSUE THREAD

**Issue by db71** [SuperJMN/WoA-Installer#12]
When do we need to install the GPU driver, before installation of Windows 10 or after?

It's experimental. What can you tell about stability?

And what performance do we gain with the GPU driver?

**Comment by CornyjK** [comment]
I don't think this is the right place for this question, we are supposed to discuss flaws and possible improvements of WoA installer here, not the drivers, we have Telegram group for that.

**Comment by db71** [comment]
Ok. Thanks. How is the group called?

**Comment by CornyjK** [comment]
@db71 
What about reading readme before posting issuses?
